build-pipeline/dotnet-framework-docker-windows-1709-amd64-images.json

```diff
@@ -182,27 +66,21 @@
       "value": "false",
       "allowOverride": true
     },
-    "image-builder.imageName": {
-      "value": "microsoft/dotnet-buildtools-prereqs:image-builder-nanoserver-20180111160403"
-    },
     "image-builder.args": {
```

Comment:
The naming on `image-builder.args` and `PB_image-builder_customArgs` is confusing in the context of passing them to the `ImageBuilderCustomArgs` arg of the build-and-test script.  What do you think about renaming `image-builder.args` to `image-builder.customArgs` and then renaming `PB_image-builder_customCommonArgs` to `PB_image-builder_queueCustomArgs`?